### ISSUE TITLE
Default upload location for Assets

### ISSUE THREAD

**Issue by pl-mnm** [verbb/hyper#101]
### What are you trying to do?

The _Asset_ link type would greatly benefit from having a default upload location. Currently when adding an asset the users are shown the root of the selected volume, but in our experience authors never create the right folder structure to cleanly order their assets, and shove everything at the root.

Our data models often require one document belonging to one entry, and we'd like the folder path to reflect that.

### What's your proposed solution?

In the _Asset_ link settings, there could be a _Default Upload Location_ field, which could include variables like `{slug}`, and the option to upload directly to the field. Similarly to Craft, users would be a offered to _Add an assset_ or  _Upload or file_.

Alternatively, perhaps we could have the ability to select a custom native Assets field in place of the default field.

**Comment by engram-design** [comment]
Agreed, and added to our todo list! There are a whole ream of other settings to include as well 
<img width="865" alt="image" src="https://github.com/verbb/hyper/assets/1221575/2398daf0-a27e-43d6-863f-d4a9e73dca2b">

**Comment by engram-design** [comment]
Updated in [3.0.0-beta.1](https://github.com/verbb/hyper/releases/tag/3.0.0-beta.1).
